test/pummel/test-dtrace-jsstack.js

```diff
@@ -86,13 +86,13 @@ dtrace.on('exit', function(code) {
   for (let i = 0; i < lines.length; i++) {
     const line = lines[i];
 
-    if (line.indexOf(sentinel) === -1 || frames.length === 0)
+    if (!line.includes(sentinel) || frames.length === 0)
       continue;
 
     const frame = line.substr(line.indexOf(sentinel) + sentinel.length);
     const top = frames.shift();
 
-    assert.strictEqual(frame.indexOf(top), 0,
+    assert.ok(frame.startsWith(top) === true,
                        `unexpected frame where ${top} was expected`);
```

Comment:
It is now linter issue: ` 96:24  error  Expected indentation of 14 spaces but found 23  indent-legacy`.